packages/migrate/src/commands/DbSeed.ts

```diff
@@ -78,9 +70,6 @@ ${chalk.bold('Options')}
       )
     }
 
-    // Print warning if user has a "ts-node" script in their package.json, not supported anymore
-    await legacyTsNodeScriptWarning()
-
```

Comment:
Only show warning if "ts-node" script is found and using `--preview-feature` or not using current "seed" configuration and have old "seed.js/ts/sh" 

To avoid warning new users who have a ts-node script (for whatever reason) and start using db seed now, they should not be warned about this.